src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -5891,7 +5891,7 @@ void FixupDispatcherContext(DISPATCHER_CONTEXT* pDispatcherContext, CONTEXT* pCo
 
     pDispatcherContext->ControlPc = (UINT_PTR) GetIP(pDispatcherContext->ContextRecord);
 
-#if defined(TARGET_ARM) || defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
+#if defined(TARGET_ARM64)
```

Comment:
Nit: It would be nice to change the ifdef around this method to `#ifdef TARGET_WINDOWS` to match the rest